rebench/executor.py

```diff
@@ -106,14 +106,19 @@ def execute(self):
 
 
 class BatchScheduler(RunScheduler):
-
     def _process_remaining_runs(self, runs):
-        for run_id in runs:
+        remaining_runs = list(runs)
+        while len(remaining_runs) > 0:
+            run_id = remaining_runs[0]
+            remaining_runs.pop(0)
```

Comment:
This will work, too: 🙂 

```suggestion
            run_id = remaining_runs.pop(0)
```